src/CMakeLists.txt

```diff
@@ -733,6 +733,15 @@ add_subdirectory(erasure-code)
 
 # Support/Tools
 if(WITH_TESTS)
+  find_program(CMAKE_MEMORYCHECK_COMMAND valgrind)
+  message(STATUS "Executing test suite with Valgrind (${CMAKE_MEMORYCHECK_COMMAND})")
+  set(MEMORYCHECK_COMMAND_OPTIONS
+  "--trace-children=yes --leak-check=full --soname-synonyms=somalloc=\*tcmalloc\* \
```

Comment:
Hmm, is overriding `--trace-children` (first `yes`, then `no`) intentional?

Reply:
no, fixing